tap_learndash/streams.py

```diff
@@ -78,15 +78,23 @@ def get_child_context(self, record: dict, context: Optional[dict]) -> dict:
 class UsersStream(LearnDashStream):
     """Defines all the fields that exist within the wordpress user record."""
     name = "users"
-    path = "/users"
+    path = "/users?context=edit"
```

Comment:
It would be most Pythonic for this to be defined by overwriting `get_url_params` rather than embedded directly in the URL path here. But, I will leave it to you

Reply:
But then it will be the only one without a path = variable and that will make me unsymmetrical and sad :(